tests/Api/Responses/Expected/shop/order/get_order_item_response.json

```diff
@@ -0,0 +1,103 @@
+{
+    "@context": "\/api\/v2\/contexts\/OrderItem",
+    "@id": "\/api\/v2\/shop\/order-items\/@integer@",
+    "@type": "OrderItem",
+    "variant": "\/api\/v2\/shop\/product-variants\/MUG_BLUE",
+    "productName": "Mug",
+    "id": @integer@,
+    "order": {
```

Comment:
I'm not sure if the order item response should have serialized the entire order. IMO it should focus on information about the item itself and there should be only IRI reference to the order